text/0079-cdk-2.0.md

```diff
@@ -262,9 +262,9 @@ will naturally retain the ability to opt out of this feature.
 `aws-cdk-lib` will be released to the various package managers under the
 following package name.
 
-| Github & NPM  | Maven Central                | PyPI          | NuGet            |
-| ------------- | ---------------------------- | ------------- | ---------------- |
-| `aws-cdk-lib` | `software.amazon.awscdk.lib` | `aws-cdk-lib` | `Amazon.CDK.Lib` |
+| Github & NPM  | Maven Central                | PyPI          | NuGet            | Go                                 |
+| ------------- | ---------------------------- | ------------- | ---------------- |------------------------------------|
+| `aws-cdk-lib` | `software.amazon.awscdk.lib` | `aws-cdk-lib` | `Amazon.CDK.Lib` | `github.com/aws/aws-cdk-go/awscdk` |
```

Comment:
Should we keep this consistent with the rest and suffix `lib`?

```suggestion
| `aws-cdk-lib` | `software.amazon.awscdk.lib` | `aws-cdk-lib` | `Amazon.CDK.Lib` | `github.com/aws/aws-cdk-go/awscdk-lib` |
```

Reply:
I don't see a good reason (especially given we decided to change Python as well). In Go, A hyphen can't be used in a package name and we wanted the package name to match the last part of the import name.